README.md

```diff
@@ -28,10 +28,12 @@ steps:
 
 ## Configuration
 
-### `credentials_file` (Required, string)
+### `credentials_file` (Optional, string)
 
 The file path of a Google Cloud credentials file, which is used to access the secrets. The account of the credentials file needs to have the Secret Manager Secret Accessor role (`roles/secretmanager.secretAccessor`).
 
+If not specified, the application default credential is expected to be available and used.
```

Comment:
Can you please add a link to the docs mentioning the default credentials?

Reply:
Done. Please have another look.